src/main/java/dk/kb/storage/api/v1/impl/DsStorageApiServiceImpl.java

```diff
@@ -315,6 +316,14 @@ public MappingDto getMapping(String referenceId) {
     public Integer updateKalturaIdForRecords() {
         log.debug("updateKalturaIdForRecords() called with call details: {}", getCallDetails());
         return DsStorageFacade.updateKalturaIdForRecords();        
-    }  
+    }
+    
+        
+    @Override
+    public List<DsRecordReferenceIdDto> referenceIds(String origin, Integer batchsize, Long mTime) {    
+        log.debug("referenceIds called with call details: {}", getCallDetails());
+        return DsStorageFacade.getReferenceIds(origin,mTime,batchsize);
+    }
```

Comment:
Please add a tiny java doc for this method

Reply:
Done